core/src/latest_unprocessed_votes.rs

```diff
@@ -56,7 +54,9 @@ impl LatestValidatorVotePacket {
 
         match limited_deserialize::<VoteInstruction>(&instruction.data) {
             Ok(VoteInstruction::UpdateVoteState(vote_state_update))
-            | Ok(VoteInstruction::UpdateVoteStateSwitch(vote_state_update, _)) => {
+            | Ok(VoteInstruction::UpdateVoteStateSwitch(vote_state_update, _))
+            | Ok(VoteInstruction::CompactUpdateVoteState(vote_state_update))
+            | Ok(VoteInstruction::CompactUpdateVoteStateSwitch(vote_state_update, _)) => {
```

Comment:
Can we introduce a `is_single_vote_update()` on `VoteInstruction` that returns `true/false` for each of these enum variations? That way we don't miss these cases in the future.

Can be a separate PR as well, since this seems like a bug in the existing code

Reply:
will address in subsequent pr